packages/shared/src/components/MainLayout.tsx

```diff
@@ -152,7 +153,8 @@ export default function MainLayout({
   const shouldRedirectOnboarding =
     !user &&
     isPageReady &&
-    onboardingV2 !== OnboardingV2.Control &&
+    (onboardingV2 !== OnboardingV2.Control ||
+      onboardingV3 !== OnboardingV3.Control) &&
```

Comment:
@idoshamun you're talking about here? So remove the V3 check? Because to have V3 you also have V2?

Reply:
Yes, exactly

Reply:
Removed.